docs/spec/spans/rum_v3_span.json

```diff
@@ -10,22 +9,9 @@
                     "type": "string",
                     "maxLength": 1024
                 },
-                "transaction_id": {
-                    "type": [
-                        "string",
-                        "null"
-                    ],
-                    "description": "Hex encoded 64 random bits ID of the correlated transaction.",
-                    "maxLength": 1024
-                },
-                "trace_id": {
-                    "description": "Hex encoded 128 random bits ID of the correlated trace.",
-                    "type": "string",
-                    "maxLength": 1024
-                },
-                "parent_id": {
-                    "description": "Hex encoded 64 random bits ID of the parent transaction or span.",
-                    "type": "string",
+                "pid": {
+                    "description": "Hex encoded 64 random bits ID of the parent span. Absent when the parent is a transaction.",
```

Comment:
Did we consider making this an index into the spans list, like in the v1 protocol? That would be even more compact.

Reply:
No, good idea :+1: